packages/client/src/objectSet/ObjectSetListenerWebsocket.ts

```diff
@@ -238,41 +217,22 @@ export class ObjectSetListenerWebsocket {
     const ontologyRid = await this.#client.ontologyRid;
 
     try {
-      const [temporaryObjectSet] = await Promise.all([
-        // create a time-bounded object set representation for watching
-        this.#createTemporaryObjectSet(sub.objectSet),
-
-        this.#ensureWebsocket(),
-
-        // look up the object type's rid and ensure that we have enabled object set watcher for that rid
-        // TODO ???
-        getObjectSetBaseType(sub.objectSet).then(baseType =>
-          OntologiesV2.ObjectTypesV2.get(
-            this.#client,
-            ontologyRid,
-            baseType,
-          )
-        ).then(
-          objectType => this.#enableObjectSetsWatcher([objectType.rid]),
-        ),
-      ]);
+      await this.#ensureWebsocket();
 
       // the consumer may have already unsubscribed before we are ready to request a subscription
       // so we have to acquire the pendingSubscription after the await.
       if (subscriptionIsDone(sub)) {
         return;
       }
-
       // Use new temporary object set id
-      sub.temporaryObjectSetId = temporaryObjectSet.objectSetRid;
```

Comment:
man getting all this temporary object set stuff to work was annoying. so glad you get to delete.